src/components/renderer/form-button.vue

```diff
@@ -28,6 +33,11 @@ export default {
       }
     },
     click() {
+      if (this.event === 'script') {
+        this.countExecutionScript++;
+        this.$emit('input', this.countExecutionScript);
```

Comment:
If execution enters this `if` block, then the component will end up emitting twice, once here, and once below where it calls `this.$emit(this.event, this.eventData)`. Is this intentional?